### DISCUSSION TITLE
Simple dependencies for Tails OS

### DISCUSSION THREAD

**Discussion by yanone** [1200wd/bitcoinlib#314] [Q&A]
Would you be open to find dependencies that don't need compilation?
My out-of-the-box Tails OS choked on compiling fastecdsa because the build requirements aren't installed. Since Tails is started from scratch and software needs to be installed every time, it's too cumbersome to install everything every time.

So I want to ask if you could maybe find a pure Python replacement for fastecdsa and have fastecdsa install optionally through a bitcoinlib[fast] switch?

**Comment by mccwdev** [comment]
You can use the pure python ecdsa library instead: https://pypi.org/project/ecdsa/
